samtranslator/swagger/swagger.py

```diff
@@ -1265,6 +1301,26 @@ def get_path_without_trailing_slash(path):
         # convert greedy paths to such as {greedy+}, {proxy+} to "*"
         return re.sub(r"{([a-zA-Z0-9._-]+|[a-zA-Z0-9._-]+\+|proxy\+)}", "*", path)
 
+    @staticmethod
+    def get_validator_name(validate_body, validate_request):
+        """
+        Get a readable path name to use as validator name
+
+        :param boolean validate_body: Boolean if validate body
+        :param boolean validate_request: Boolean if validate request
+        :return string: Normalized validator name
+        """
+        if validate_body and validate_request:
+            return "FULL"
```

Comment:
I changed the names. (Namimg things are hard). I did on uppercase because I saw somewhere on the comments it being uppercase so I stick with ir. But all docs refer as lowercase you are right, fixed it!